### PR TITLE
Standardize code blocks

### PR DESCRIPTION
**Pull request by andresriveratoro**
https://vajira.max.gov/browse/API-24058

### CODE REVIEW

File: src/containers/documentation/swaggerPlugins/WrapHighlightCode.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+import { CodeBlock, CodeWrapper } from '../../../components';
 
 export const WrapHighlightCode = {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   highlightCode: (Original: React.ComponentType): React.ComponentType => {
     const HighlightCode = (props: Record<string, unknown>): JSX.Element => (
       <CodeWrapper>
-        <Original {...props} />
+        <CodeBlock withCopyButton code={props.value as string} language={props.language as string} />
       </CodeWrapper>
```
Comment (imcgonigle) [inline src/containers/documentation/swaggerPlugins/WrapHighlightCode.tsx:11]: Can the CodeWrapper component be removed from this file or would that break something? The CodeBlock component uses CodeWrapper so the one in this file could probably be removed.

Comment (andresriveratoro) [reply src/containers/documentation/swaggerPlugins/WrapHighlightCode.tsx:11]: Good catch!

**Review by imcgonigle — DISMISSED** [review]
Looks good to me

**Review by Cgfor3 — APPROVED** [review]
🚢

**Pull request merged**